module/os/windows/zfs/zvol_os.c

```diff
@@ -1035,3 +1035,87 @@ zvol_fini(void)
 	zvol_fini_impl();
 	taskq_destroy(zvol_taskq);
 }
+
+void
+get_zpool_name(PDEVICE_OBJECT DeviceObject, char *zpool_name,
+    size_t size) {
```

Comment:
If this method is not used then pls remove.

Reply:
done